lib/pure/json.nim

```diff
@@ -363,10 +392,25 @@ proc `[]=`*(obj: JsonNode, key: string, val: JsonNode) {.inline.} =
   assert(obj.kind == JObject)
   obj.fields[key] = val
 
+template noserialize*() {.pragma.}
+
 proc `%`*[T: object](o: T): JsonNode =
   ## Construct JsonNode from tuples and objects.
+  ## An object field annotated with the `{.noserialize.}` pragma will not appear
```

Comment:
```suggestion
  ##
  ## An object field annotated with the `{.noserialize.}` pragma will not appear
```

RST rendering will not put a line break unless you include an empty line like I have done above

Reply:
Oh, thanks! I'm not very experienced with RST.